src/kontrol/deployment.py

```diff
@@ -1,22 +1,33 @@
 from dataclasses import dataclass
 from typing import NamedTuple
 
+from web3 import Web3
 
```

Comment:
I think an empty line after the import is missing, `make format` should add it automatically.

Reply:
Oh I don't know how I missed that one! Thanks!